src/blueprints/tv2/timeline-object-factories/tv2-caspar-cg-timeline-object-factory.ts

```diff
@@ -204,21 +207,42 @@ export class Tv2CasparCgTimelineObjectFactory implements Tv2GraphicsElementTimel
     }
   }
 
-  public createSplitScreenLocatorTimelineObject(): CasparCgTemplateTimelineObject {
-    return {} as CasparCgTemplateTimelineObject
-    // TODO: SOF-1695 Should implement this
-    // return {
-    //   id: 'casparCg_locators',
-    //   enable: {
-    //     start: 0
-    //   },
-    //   priority: 1,
-    //   layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
-    //   content: {
-    //     deviceType: DeviceType.CASPAR_CG,
-    //     type: CasparCgType.TEMPLATE
-    //   }
-    // }
+  public createSplitScreenLocatorTimelineObject(graphicsSetup: GraphicsSetup, splitScreenConfiguration: SplitScreenConfiguration, locatorLabels?: string[]): CasparCgTemplateTimelineObject {
+    return {
+      id: 'casparCg_locators',
+      enable: {
+        start: 0
+      },
+      priority: 1,
+      layer: Tv2GraphicsLayer.GRAPHICS_LOCATORS,
+      content: {
+        deviceType: DeviceType.CASPAR_CG,
+        type: CasparCgType.TEMPLATE,
+        templateType: 'html',
+        name: `${graphicsSetup.htmlPackageFolder}/${HTML_GRAPHICS_INDEX_FILENAME}`,
+        useStopCommand: false,
+        mixer: {
+          opacity: 100
+        },
+        data: {
+          display: 'program',
+          partialUpdate: true,
+          slots:
+            {
+              '850_dve': {
```

Comment:
I think this should make use of `this.mapTv2GraphicsLayerToHtmlGraphicsSlot(Tv2GraphicsLayer.GRAPHICS_LOCATORS)` to define the used slot similarly to other methods in this class that return `CasparCgTemplateTimelineObject`.

Doing so would also ensure we make use of the `CasparCgSlot` Enum, instead of a magic string.

Reply:
It's not a magic string.
`850_dve` is defined on `CasparCgTemplateData`.
![image](https://github.com/tv2/sofie-server/assets/23049647/4bb3cdb4-2b21-413e-9695-a4ab90f19252)
Writing something else result in a compile error:
![image](https://github.com/tv2/sofie-server/assets/23049647/d5886658-a8d5-4019-9bc6-6e759420e67c)

Reply:
I see.

Perhabs it is the other way around then, and where we use `this.mapTv2GraphicsLayerToHtmlGraphicsSlot` we should not, and instead have it defined as done in this method.

Reply:
Whether or not it is a magic string depends on your definition of a magic string – which there are multiple variations of. But the literal string is typed. Personally I think it would be nice to use an enumeration to indicate what slots we support and give them meaningful names. E.g. instead of `850_dve`, we could have the value `CasparCgSlot.SPLIT_SCREEN`. This way, we also reduce the co-existence of DVE and Split screen in the code.

Currently we mix the typed string literals and the `CasparCGSlot` enum for types and values. The enum is only used (partially) in this file. 
I'll suggest having the enum moved out and use it to define the supported slots on the `CasparCgTemplateData` interface.

Reply:
In addition, the `display` and `payload.type` attributes currently have the type `string`, but only support specific values. That would also be nice to have specialized.